### PR TITLE
[OP#48444] Make changes according to figma in UI

### PR DESCRIPTION
**Pull request by SagarGi**
### Description
This PR makes changes according to figma in the UI.
figma : https://www.figma.com/file/gtLQfPe09X7XugAH8L7dTy/Nextcloud-Integration?type=design&node-id=3663-101659

### Related WP
https://community.openproject.org/projects/817/work_packages/48444/activity

### CODE REVIEW

File: src/components/AdminSettings.vue [hunk 1]
```diff
@@ -500,7 +500,12 @@ export default {
 		userAppPasswordHint() {
 			const linkText = t('integration_openproject', 'Administration > File storages')
 			const htmlLink = `<a class="link" href="${this.adminFileStorageHref}" target="_blank" title="${linkText}">${linkText}</a>`
-			return t('integration_openproject', 'This value is only visible now and then never again. Copy this password to OpenProject {htmlLink} as an Administrator.', { htmlLink }, null, { escape: false, sanitize: false })
+			return t('integration_openproject', 'This value will only be accessible once. Now, as an administrator copy this password to OpenProject {htmlLink}.', { htmlLink }, null, { escape: false, sanitize: false })
+		},
+		errorHintForGroupFolderNameAlreadyExists() {
+			const linkText = t('integration_openproject', 'troubleshooting guide')
+			const htmlLink = `<a class="link" href="/404" target="_blank" title="${linkText}">${linkText}</a>`
```
Comment (SwikritiT) [inline src/components/AdminSettings.vue:507]:
```suggestion
			const htmlLink = `<a class="link" href="https://www.openproject.org/docs/system-admin-guide/integrations/nextcloud/" target="_blank" title="${linkText}">${linkText}</a>`
```

I think we can use this link here, the help guide will probably be added here

Comment (SagarGi) [reply src/components/AdminSettings.vue:507]: sure

File: src/components/AdminSettings.vue [hunk 2]
```diff
@@ -580,10 +585,10 @@ export default {
...
 		projectFolderSetUpErrorMessageDescription(errorKey) {
 			switch (errorKey) {
-			case 'The group folder name "OpenProject" integration already exists' :
-				return t('integration_openproject', 'Please make sure to rename the group folder or completely delete the previous one or deactivate the automatically managed folders.')
-			case 'The group folder app is not installed' :
-				return t('integration_openproject', 'Please install the group folder to be able to use automatic managed folders or deactivate the automatically managed folders.')
+			case 'The group folder name "OpenProject" already exists' :
+				return this.errorHintForGroupFolderNameAlreadyExists
+			case 'The "Group folders" app is not installed' :
+				return t('integration_openproject', 'Please install the "Group folders" to be able to use automatic managed folders or deactivate the automatically managed folders.')
```
Comment (SwikritiT) [inline src/components/AdminSettings.vue:591]:
```suggestion
				return t('integration_openproject', 'Please install the "Group folders" app to be able to use automatic managed folders or deactivate the automatically managed folders.')
```

**Comment by SwikritiT** [comment]
@SagarGi I think in this case and the case where it shows that the "Openproject user already exists"
Screenshot from 2023-06-08 10-49-02

we need to show the error message that directs toward the documentation link.

**Comment by SagarGi** [comment]
> @SagarGi I think in this case and the case where it shows that the "Openproject user already exists" Screenshot from 2023-06-08 10-49-02
> 
> we need to show the error message that directs toward the documentation link.

For now i have put the same for the user, group and groupname error. we can aske this in meeting.

File: src/components/AdminSettings.vue [hunk 3]
```diff
@@ -580,16 +585,10 @@ export default {
...
-			case 'The group folder name "OpenProject" integration already exists' :
-				return t('integration_openproject', 'Please make sure to rename the group folder or completely delete the previous one or deactivate the automatically managed folders.')
-			case 'The group folder app is not installed' :
-				return t('integration_openproject', 'Please install the group folder to be able to use automatic managed folders or deactivate the automatically managed folders.')
-			case 'The user "OpenProject" already exists' :
-				return t('integration_openproject', 'Please make sure to completely delete the previous user or deactivate the automatically managed folders.')
-			case 'The group "OpenProject" already exists' :
-				return t('integration_openproject', 'Please make sure to completely delete the previous group or deactivate the automatically managed folders.')
+			case 'The "Group folders" app is not installed' :
+				return t('integration_openproject', 'Please install the "Group folders" app to be able to use automatic managed folders or deactivate the automatically managed folders.')
```
Comment (SwikritiT) [inline src/components/AdminSettings.vue:589]: I think this should be `automatically`
```suggestion
				return t('integration_openproject', 'Please install the "Group folders" app to be able to use automatically managed folders or deactivate the automatically managed folders.')
```

File: tests/jest/components/AdminSettings.spec.js [hunk 4]
```diff
@@ -969,31 +969,31 @@ describe('AdminSettings.vue', () => {
...
+									error: 'The "Group folders" app is not installed',
+									expectedErrorDetailsMessage: 'Please install the "Group folders" app to be able to use automatic managed folders or deactivate the automatically managed folders.',
 								},
 							],
 							[
 								'should set the user already exists error message and error details when user already exists',
 								{
 									error: 'The user "OpenProject" already exists',
-									expectedErrorDetailsMessage: 'Please make sure to completely delete the previous user or deactivate the automatically managed folders.',
+									expectedErrorDetailsMessage: 'Setting up the OpenProject user, group and group folder was not possible. Please check this troubleshooting guide on how to resolve this situation.',
```
Comment (SwikritiT) [inline tests/jest/components/AdminSettings.spec.js:982]: The error message is the same so maybe we can add it in a variable somewhere and can reuse it?
```suggestion
									expectedErrorDetailsMessage: groupFolderSetupErrorMessage
```

Comment (SwikritiT) [reply tests/jest/components/AdminSettings.spec.js:982]: I think we can make a separate file containing all the error messages and use it in `vue` files and also in `specs`. This should not be in this PR so I'm resolving this for now. I'll create a separate work-packages

**Review by SwikritiT — APPROVED** [review]
LGTM

File: src/components/AdminSettings.vue (hunk 3 above)
Comment (SwikritiT) [reply src/components/AdminSettings.vue:589]: let's merge this PR we can make this change when we get information from other side

**Pull request merged**